ports/libopenmpt/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "libopenmpt",
-  "version": "0.5.12",
-  "description": "a library to render tracker music",
+  "version": "0.6.7",
+  "description": "a cross-platform C++ and C library to decode tracked music files (modules) into a raw PCM audio stream.",
   "homepage": "https://openmpt.org/",
```

```suggestion
  "version": "0.6.7",
  "description": "a cross-platform C++ and C library to decode tracked music files (modules) into a raw PCM audio stream.",
  "homepage": "https://openmpt.org/",
  "license": "BSD-3-Clause",
```

A `license` is required.